code/modules/spells/spell_types/list_target/telepathy.dm

```diff
@@ -38,6 +38,7 @@
 
 	to_chat(owner, "<span class='[bold_telepathy_span]'>You transmit to [cast_on]:</span> [formatted_message]")
 	if(!cast_on.can_block_magic(antimagic_flags, charge_cost = 0)) //hear no evil
+		balloon_alert(cast_on, "voice in your head!")
```

Comment:
maybe we can instead say "something whispers in your ear" or "you hear something", though it would seem out of place when compared to other balloon alerts

Reply:
it's a voice in your head